content/articles/cloud-management/index.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: engineering-education
+title: Cloud Management
+description: Cloud management is the control of products and services running in the cloud. It caters for these products and services. In this article we will get to understand how to choose the best cloud platforms. Also, we'll discuss challenges of cloud management and ways to beat them.
+author: anyebe-blessing-ene
+date: 2021-08-06T00:00:00-10:30
+topics: {Cloud}
+excerpt_seperator: <!--more-->
+images: 
+---
+In this article, we will discuss how cloud management and understand what there is to know about it.
```

Comment:
You could start with a more descriptive introduction.

Why does the reader want to know about cloud management? What is it? How will it help the reader?  Then, share the goals/objectives of this tutorial

You must make the reader feel interested about this article.

Reply:
Start with cloud. Explain how apps migrated from local to cloud.